lib/network/modules/components/edges/BezierEdgeStatic.js

```diff
@@ -4,15 +4,14 @@ import BezierEdgeBase from './util/BezierEdgeBase'
  * A Static Bezier Edge. Bezier curves are used to model smooth gradual
  * curves in paths between nodes.
  *
- * @class BezierEdgeStatic
+ * @param {Object} options
+ * @param {Object} body
+ * @param {Label} labelModule
  * @extends BezierEdgeBase
  */
 class BezierEdgeStatic extends BezierEdgeBase {
   /**
-   * @param {Object} options
-   * @param {Object} body
-   * @param {Label} labelModule
-   * @constructor BezierEdgeStatic
+   * @inheritDoc
```

Comment:
Hah! This one's valid. There is a parent class.
But why here? It would really expect it to be in the class header.

Reply:
It's inheriting from the encapsulating class, since the class and constructor are equivalent in terms of params, etc.

Reply:
OK, I guess. TIL. But I would think that with the class-syntax the logical place would be to put it on top of the class.

But if it works, it works.